tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/free_text_survey_response_suppression_test.py

```diff
@@ -0,0 +1,206 @@
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
+
+        cls.rule_instance = FreeTextSurveyResponseSuppression(
+            project_id, dataset_id, sandbox_id)
+
+        cls.fq_table_names = [
+            f'{project_id}.{dataset_id}.{OBSERVATION}',
+        ]
+
+        for table_name in [OBSERVATION]:
+            sandbox_table_name = cls.rule_instance.sandbox_table_for(table_name)
+            cls.fq_sandbox_table_names.append(
+                f'{cls.project_id}.{cls.sandbox_id}.{sandbox_table_name}')
+
+        # Add SUPPRESSION_RULE_CONCEPT_TABLE to fq_sandbox_table_names so it gets deleted after
+        # the test
+        cls.fq_sandbox_table_names.append(
+            f'{cls.project_id}.{cls.sandbox_id}.{SUPPRESSION_RULE_CONCEPT_TABLE}'
+        )
+
+        # call super to set up the client, create datasets
+        cls.up_class = super().setUpClass()
+
+    def setUp(self):
+        """
+        Create common information for tests.
+
+        Creates common expected parameter types from cleaned tables and a common
+        fully qualified (fq) dataset name string used to load the data.
+        """
+        self.observation_date = parser.parse('2017-05-02').date()
+
+        fq_dataset_name = self.fq_table_names[0].split('.')
+        self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
+
+        fq_sandbox_name = self.fq_sandbox_table_names[0].split('.')
+        self.fq_sandbox_name = '.'.join(fq_sandbox_name[:-1])
+
+        super().setUp()
+
+    def test_free_text_suppression_cleaning(self):
+        """
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.
+        """
+
+        concept_table_tmpl = self.jinja_env.from_string("""
+            DROP TABLE IF EXISTS `{{fq_dataset_name}}.concept`;
+            CREATE TABLE `{{fq_dataset_name}}.concept` 
+            AS (
+            WITH w AS (
+                SELECT ARRAY<STRUCT<
+                    concept_id INT64,
+                    concept_name STRING,
+                    domain_id STRING,
+                    vocabulary_id STRING,
+                    concept_class_id STRING,
+                    concept_code STRING,
+                    valid_start_date STRING,
+                    valid_end_date STRING
+                    >>
+            [(111, 'Something Text Box', 'Observation', 'SNOWMED', 'Context-dependent', 'WhiteFreeText', 
+                    '2016-05-01', '2016-05-02'),
+                (222, 'TextBox', 'Observation', 'SNOWMED', 'Context-dependent', 'BlackFreeText', 
+                    '2016-05-01', '2016-05-02'),
```

Comment:
It's not concept_name that I was suggesting to change.  You're rule is looking at `concept_codes`.  I thought it would be good to test having `TextBox` in a concept_code.

```suggestion
                (222, 'Text Box', 'Observation', 'SNOWMED', 'Context-dependent', 'ATextBox', 
                    '2016-05-01', '2016-05-02'),
```

Reply:
Oh, gotcha! 😅